lib/staleBuildChecks.js

```diff
@@ -63,19 +54,10 @@ const removeOldBuildLabel = async (context) => {
   if (!containsOldBuildLabel) {
     return;
   }
-
-  // Get last commit for pull request.
-  const {data: lastCommit} = await context.github.repos.getCommit(
-    context.repo({
-      ref: pullRequest.head.sha
-    })
-  );
-
-  const lastCommitDate = new Date(lastCommit.commit.author.date);
   // Commit is older than 2 days, this can happen when 'synchronize' event
   // got triggered as a result of a merge to the main branch. In this case
   // we don't want to remove the label.
-  if (utils.MIN_BUILD_DATE > lastCommitDate) {
+  if (utils.checkBuild(pullRequest, context)) {
```

Comment:
There's no `utils.checkBuild` function.

Reply:
Extremely sorry sir for such mistake fixing it in latest commit.